plugins/modules/dcnm_network.py

```diff
@@ -20,7 +20,7 @@
 import copy
```

Comment:
Please add doc header comments to indicate that for multisite fabrics, the network can only be configured on the multisite fabric, not the member fabrics.  A similar message was added to the `dcnm_vrf` module.

See: https://github.com/CiscoDevNet/ansible-dcnm/pull/48/files#diff-ec02ce28cda389e26f6761cd904c08f9c6c0a87b35d368d9e7631138b19b9ffdR36